ModelBundle/Resources/translations/validators.fr.yml

```diff
@@ -4,3 +4,4 @@ open_orchestra_model_validators:
     document:
         content_type:
             unique_field_id: Les identifiants des champs doivent être unique
+            unique_content_type_id: Cette identifiant est déjà utilisé, s'il vous plaît choisissez en un autre
```

Comment:
"Cet identifiant"
and maybe invert the second part, it sounds a little google translated :p "choisissez en un autre s'il vous plait"